scripts/helpers/openai-compatible-agent.py

```
@@ -0,0 +1,95 @@
+
+def tool_exec(cwd: Path, name: str, args: dict) -> str:
+    try:
+        if name == "read_file":
+            return resolve_path(cwd, str(args.get("path", ""))).read_text(encoding="utf-8", errors="replace")[:20000]
+        if name == "write_file":
+            p = resolve_path(cwd, str(args.get("path", ""))); p.parent.mkdir(parents=True, exist_ok=True)
+            p.write_text(str(args.get("content", "")), encoding="utf-8")
+            return f"wrote {p.relative_to(cwd.resolve())} ({p.stat().st_size} bytes)"
+        if name == "run_command":
+            cmd = str(args.get("command", ""))
+            if len(cmd) > 600: return "ERROR: command too long"
```

Ruff (0.15.15)

[error] Multiple statements on one line (colon)

(E701)